xsamtools/cram.py

```diff
@@ -22,6 +22,13 @@
 CramLocation = namedtuple("CramLocation", "chr alignment_start alignment_span offset slice_offset slice_size")
 log = logging.getLogger(__name__)
 
+def file_definition(fh):
```

Comment:
Please use a more descriptive name, e.g. "decode_cram_version_header", or something.

Reply:
Also, please add parameter and return type hints, e.g. `fh: typing. BinaryIO`.